h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -119,11 +119,14 @@ function TextField({
 }
 
 export default function CreateGroupForm() {
-  const [name, setName] = useState('');
-  const [description, setDescription] = useState('');
+  const config = useMemo(() => readConfig(), []);
+  const mode = config.components.CreateEditGroupForm.mode;
+  const group = config.components.CreateEditGroupForm.group;
```

Comment:
We can probably do without the `mode` string and the component could instead react to the presence or absence of existing group details in the `js-config`